### PR TITLE
Add Dynatrace information

### PR DESCRIPTION
**Pull request by Richard87**
Needs https://github.com/equinor/radix/issues/115

### CODE REVIEW

**Review by satr — COMMENTED** [review]
Use of internal links needs to be approved by PO

File: public-site/docs/src/docs/topic-dynatrace-int/index.md [hunk 1]
```diff
@@ -6,45 +6,87 @@ title: Integrate Dynatrace in a Radix app
...
...e](https://statoilsrm.sharepoint.com/sites/applicationperformancemanagement/SitePages/Container-monitoring---attaching-to-a-management-zone.aspx?xsdata=MDV8MDF8fDQzNjU4MDg2MjJmMjQ1MDE5Njk4MDhkYmYxODE0OGYwfDNhYTRhMjM1YjZlMjQ4ZDU5MTk1N2ZjZjA1YjQ1OWIwfDB8MHw2MzgzNjkzMDk3Nzk4NTkwOTZ8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxMMk5vWVhSekx6RTVPalV6WkdVeU1EVm1MVEF3WkRRdE5HUTFZeTA0TXpZM0xUWTFOalJtTkRBd1kyWXhOVjlpWXpNMFlqQXlaQzB3WXpoaExUUXlZbVV0T1RneE1DMWlaREU0TkdFM05qSXpZMlJBZFc1eExtZGliQzV6Y0dGalpYTXZiV1Z6YzJGblpYTXZNVGN3TVRNek5ERTNOekUzT0E9PXw0MzlhNGM1MmYwZjA0OGJmOTY5ODA4ZGJmMTgxNDhmMHwwNDdlNTBlNDJiMDc0MmUwYWE0ZDBmZGE0MmI3YzQzNg%3d%3d&sdata=UFpyalVqVnh2QUxMWmpON283dlRiQ09qb1lEY2JKUGwzWUhJNzFYZnZzaz0%3d&ovuser=3aa4a235-b6e2-48d5-9195-7fcf05b459b0%2cRIHAG%40equinor.com&OR=Teams-HL&CT=1701334180996&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiI0OS8yMzExMDIzMTgwOCIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3d%3d&SafelinksUrl=https%3a%2f%2fstatoilsrm.sharepoint.com%2fsites%2fapplicationperformancemanagement%2fSitePages%2fContainer-monitoring---attaching-to-a-management-zone.aspx).
```
Comment (emirgens) [inline public-site/docs/src/docs/topic-dynatrace-int/index.md:91]:
```suggestion
- Read about Dynatrace Container monitoring integration [here](https://statoilsrm.sharepoint.com/sites/applicationperformancemanagement/SitePages/Container-monitoring---attaching-to-a-management-zone.aspx)
```

File: public-site/docs/src/docs/topic-dynatrace-int/index.md [hunk 2]
```diff
@@ -6,45 +6,87 @@ title: Integrate Dynatrace in a Radix app
...
+- Read about Dynatrace secrets and configuration [here](https://statoilsrm.sharepoint.com/sites/applicationperformancemanagement/SitePages/Install-on-Linux.aspx?xsdata=MDV8MDF8fDRlMTgyMmRkYTU3YjRmOWVmNGRiMDhkYmYxODE1NmYzfDNhYTRhMjM1YjZlMjQ4ZDU5MTk1N2ZjZjA1YjQ1OWIwfDB8MHw2MzgzNjkzMTAwMTUxMTcwNTl8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxMMk5vWVhSekx6RTVPalV6WkdVeU1EVm1MVEF3WkRRdE5HUTFZeTA0TXpZM0xUWTFOalJtTkRBd1kyWXhOVjlpWXpNMFlqQXlaQzB3WXpoaExUUXlZbVV0T1RneE1DMWlaREU0TkdFM05qSXpZMlJBZFc1eExtZGliQzV6Y0dGalpYTXZiV1Z6YzJGblpYTXZNVGN3TVRNek5ESXdNRE14TWc9PXw5MGI0NGI0YzE1NDQ0OTBlZjRkYjA4ZGJmMTgxNTZmM3w0NDg2ODMxNDBhODA0YWFmOTk2Zjk1MGEwODllNDdkYQ%3D%3D&sdata=Qm1TMFRaV2d1YVo1SkU2QkpIUVpDYVpPTTVQMEJnajlTdUorM2ZLYzdsUT0%3D&ovuser=3aa4a235-b6e2-48d5-9195-7fcf05b459b0%2CRIHAG%40equinor.com&OR=Teams-HL&CT=1701334205387&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiI0OS8yMzExMDIzMTgwOCIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D).
```
Comment (emirgens) [inline public-site/docs/src/docs/topic-dynatrace-int/index.md:92]:
```suggestion
- Read about Dynatrace secrets and configuration [here](https://statoilsrm.sharepoint.com/sites/applicationperformancemanagement/SitePages/Install-on-Linux.aspx)
```

**Review by emirgens — CHANGES_REQUESTED** [review]
Now I see the issue, these links will work without the token. Then the access is "protected"

**Review by emirgens — APPROVED** [review]
ok

**Pull request merged**